src/components/asset-list/RecyclerAssetList2/core/useMemoBriefSectionData.ts

```diff
@@ -95,8 +95,7 @@ export default function useMemoBriefSectionData({
 
         if (
           data.type === CellType.PROFILE_STICKY_HEADER ||
-          data.type === CellType.NFTS_HEADER ||
-          data.type === CellType.POSITIONS_HEADER
+          data.type === CellType.NFTS_HEADER
```

Comment:
havnt tested so may be handled somewhere else but I think we can get ride of the NFT_HEADER being sticky as well